### PR TITLE
Issue #657: allow to run only one feature file

### PR DESCRIPTION
**Pull request by tsedmik**
We will be able to run only one feature file from integration tests
(not all of them).

Signed-off-by: Tomáš Sedmík <tsedmik@redhat.com>

### CODE REVIEW

**Comment by praveenkumar** [comment]
@tsedmik Can you also put a section in the https://github.com/code-ready/crc/blob/master/developing.adoc file about running the integration tests as a separate commit in this PR so that it would be easy for us to have those details capture and use it?

**Comment by tsedmik** [comment]
@praveenkumar I moved the documentation about running integration tests into _Developing Guide_ + I added mention about how to execute only one feature file. WDYT?

File: developing.adoc [hunk 1]
```diff
@@ -5,7 +5,7 @@
 [[developing-overview]]
 == Overview
 
-The following sections describe how to build the project.
+The following sections describe how to build and test the project.
```
Comment (praveenkumar) [inline developing.adoc]: :+1:

File: developing.adoc [hunk 2]
```diff
@@ -46,3 +46,41 @@ For more information, see the following:
...
+[[running-integration-tests]]
+== Running integration tests
+
+We have automated integration tests which keep CodeReady Containers in shape.
+
+[[intro-to-integration-testing]]
+=== Introduction
+Integration tests use the link:http://github.com/code-ready/clicumber[Clicumber] package to provide basic functionality for testing CLI binaries.
+
+Clicumber allows running commands in a persistent shell instance (`bash`, `tcsh`, `zsh`, Command Prompt, or PowerShell), assert its outputs (standard output, standard error, or exit code), check configuration files, and so on. The general functionality of Clicumber is then extended by CodeReady Containers specific test code to cover the whole functionality of CodeReady Containers.
```
Comment (praveenkumar) [inline developing.adoc]: :+1:

File: developing.adoc [hunk 3]
```diff
@@ -46,3 +46,41 @@ For more information, see the following:
...
+
+===== How to run only a subset of all integration tests
+
+Implicitly, all integration tests for your operating system are executed. If you want to run only tests from one feature file, you have to override `GODOG_OPTS` environment variable. For example:
+
+```bash
+make integration GODOG_OPTS="--godog.tags='@basic && @windows'" BUNDLE_LOCATION=<bundle location> PULL_SECRET_FILE=<pull secret path>
+```
+
+Please notice `@basic && @windows`, where `@basic` tag stands for `basic.feature` file and `@windows` tag for integration tests designed for Windows.
```
Comment (praveenkumar) [inline developing.adoc]: :+1:

**Pull request merged**